### ISSUE TITLE
Bad options: --specs=nano.specs

### ISSUE THREAD

**Issue by mindThomas** [ogrebenyuk/compilerexplorer#17]
Getting error with certain compiler flags when trying to use Compiler Explorer with a Cross Compile toolchain, ARM gcc 7.3.1 (none) in this case.

**Comment by ogrebenyuk** [comment]
Yes, other users also occasionally reported bad compiler options that the backend can't handle. Having no chance to anticipate such occurrences, I have added a generic blacklist - the compiler options that are filtered out and are not sent to the remote compiler.

Go to Settings -> Build, Execution, Development -> Compiler Explorer, and ad your offending compiler options to the "Ignore compiler switches" list
